system_status/Dockerfile

```diff
@@ -1,3 +1,16 @@
-FROM public.ecr.aws/lambda/python:3.9
+# Docker image for the SNS to SQS SMS callback the lambda API.
```

Comment:
is this correct?

Reply:
I couldn't get notification-utils to work without bumping python to 3.10!

Reply:
sorry I meant the comment on. l#1